packages/cli/src/cmds/beacon/initBeaconState.ts

```diff
@@ -44,6 +44,15 @@ export async function initBeaconState(
     return await initFromFile(args.weakSubjectivityStateFile);
   } else if (dbHasSomeState) {
     return await initStateFromDb(config, db, logger);
+  } else if (args.fetchWeakSubjectivityStateFromIPFS) {
+    const state = await initFromFile("https://ipfs.io/ipfs/QmVsGxwZs4acTDfhQrNYVqccEVxyhALqJuYqgJrbbNAd1Z/state.ssz");
+    const storedStateRoot = await downloadOrLoadFile("./ws-state-root.ssz");
```

Comment:
Can we pull out the ipfs path and state root into a separate file, eg: weakSubjectivityState.json.
Something that ends up looking like:
```json
{
  "ipfsPath": "/ipfs/QmVsGxwZs4acTDfhQrNYVqccEVxyhALqJuYqgJrbbNAd1Z/state.ssz",
  "stateRoot": "0xabcd..."
}
```

I think this will be easier to manage.